src/db/tables/subscribers.js

```diff
@@ -64,13 +64,36 @@ async function getSubscriberByEmail (email) {
  * @returns {object} updated subscriber
  */
 async function updatePrimaryEmail (subscriber, updatedEmail) {
-  const updated = await knex('subscribers')
-    .where('id', '=', subscriber.id)
-    .update({
-      primary_email: updatedEmail
-    })
-    .returning('*')
-  const updatedSubscriber = Array.isArray(updated) ? updated[0] : null
+  const trx = await knex.transaction()
+  let subscriberTableUpdated, emailTableUpdated
+  try {
+    // update subscriber primary email to updated email
+    subscriberTableUpdated = await knex('subscribers')
+      .where('id', '=', subscriber.id)
+      .update({
+        primary_email: updatedEmail
+      })
+      .returning('*')
+      .transacting(trx)
+
+    // if email_addresses table has updatedEmail as a secondary in Monitor
```

Comment:
Adding fix for an edge case here:
* When we have two emails, A & B. If we swap them in FxA, we need to swap them in Monitor as well. Prior to this, we'd get a dup because only the `subscribers` table email was changed